api/main.py

```diff
@@ -100,7 +100,11 @@ async def post_node(node: Node, token: str = Depends(get_user)):
             status_code=status.HTTP_400_BAD_REQUEST,
             detail=str(e)
         )
-    await pubsub.publish_cloudevent('node', {'op': op, 'id': str(obj.id)})
+    await pubsub.publish_cloudevent('node', {
+                                    'op': op, 'id': str(obj.id),
+                                    'name': str(obj.name),
+                                    'revision': str(obj.revision)}
+                                    )
```

Comment:
Please don't change the data sent in the event, the aim is only to filter which events are being sent. This is to reduce the number of events being sent, and also to simplify or remove the need for filtering on the client side.

Reply:
However, we can pass a dictionary with the attributes and values from the object as an extra argument to the `publish_cloudevent()` method to use as the data for applying filters.

For example, the object JSON dump would contain something like `{"name": "checkout", "revision": {...}}` and that can then be used for filtering with each pub/sub subscription.